packages/web-twig/src/Resources/components/TextArea/stories/TextAreaValidation.twig

```diff
@@ -1,38 +1,37 @@
 <TextArea
-    id="textareaDanger"
-    isRequired
+    id="textareaSuccess"
     label="Label"
-    name="textareaDanger"
+    name="textareaSuccess"
     placeholder="Placeholder"
-    validationState="danger"
+    validationState="success"
+    value="Filled"
 />
 <TextArea
-    id="textareaDangerMessage"
-    isRequired
+    id="textareaWarning"
     label="Label"
-    message="Validation message"
-    name="textareaDangerMessage"
+    validationText="Validation text"
+    name="textareaWarning"
     placeholder="Placeholder"
-    validationState="danger"
+    validationState="warning"
     value="Filled"
 />
 <TextArea
-    id="textareaSuccess"
-    isRequired
+    id="textareaDanger"
     label="Label"
-    message="Validation message"
-    name="textareaSuccess"
+    validationText="{{ ['First validation text', 'Second validation text'] }}"
```

Comment:
The messages are slightly different in the web package:

![obrazek](https://github.com/lmc-eu/spirit-design-system/assets/5614085/3d065a18-5b27-4191-9039-f1f627e08cd5)